lib/galaxy_test/selenium/test_uploads.py

```diff
@@ -13,7 +13,6 @@
 class UploadsTestCase(SeleniumTestCase, UsesHistoryItemAssertions):
     @selenium_test
     def test_upload_simplest(self):
-        self.use_legacy_history()
```

Comment:
This is another example, this test implicitly exists for the new history too but in this particular case the uploaded dataset is expected in the legacy history element.